scripts/update-versions.mjs

```
@@ -0,0 +1,66 @@
+function updateVersionInRootPackageLockJsonFile(newVersion) {
+  const path = 'package-lock.json';
+  console.info('Updating', path);
+  const fileContent = readFileContent(path);
+  const packageJson = JSON.parse(fileContent);
+  packageJson.packages['packages/core'].version = newVersion;
+  writeJsonContentToFile(path, packageJson);
+}
+
+function updateVersionInCorePackageJsonFile(newVersion) {
+  const path = 'packages/core/package.json';
+  console.info('Updating', path);
```

ESLint

[error] Unexpected console statement.

(no-console)